cl/search/models.py

```diff
@@ -3225,6 +3225,12 @@ class Opinion(AbstractDateTimeModel):
         ]
     )
     ordering_key = models.IntegerField(null=True, blank=True)
+    main_version = models.ForeignKey(
+        "self",
```

Comment:
We'll want a related name value here, no, so that we can easily get to the secondary opinions from the primary?